Monika After Story/game/script-topics.rpy

```diff
@@ -17093,3 +17093,46 @@ label monika_tanabata:
     else:
         m 3eua "Even if you're not interested in travelling, it's pretty interesting to explore other cultures, isn't it?"
     return
+
+#NOTE: Unlocked after Monika reacts to reddit (I hope)
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_reddit",
+            category=['media', 'monika'],
+            prompt="Subreddits",
+            random=True
```

Comment:
With previous comment, this would make the action irrelevant, so we can just remove it.
```suggestion
```